packages/dynamite/dynamite_runtime/pubspec.yaml

```diff
@@ -22,3 +29,11 @@ dev_dependencies:
       url: https://github.com/nextcloud/neon
       path: packages/neon_lints
   test: ^1.24.9
+
+platforms:
+  windows:
+  linux:
+  macos:
+  android:
+  ios:
+  web:
```

Comment:
I think this should be further up, before or after the environment

Reply:
I prefer not to scroll to get to the relevant information (deps and environment).
But am open if you want it further up.

Reply:
Good point